### PR TITLE
Refactor README.md

### PR DESCRIPTION
**Pull request by gpotter2**
This PR:
- removes "Hands on" from the README. IMH it's better to link to the doc introduction instead
- simplify it. We want to be direct: it's a README
- links in https

Demo: https://github.com/gpotter2/scapy/blob/gallery/README.md

**Requires https://github.com/secdev/secdev.github.io/pull/20**

### CODE REVIEW

**Comment by guedou** [comment]
We can improve it, but I really prefer the way it is. I think that it is really convenient to be able to test a project by reading the README, and to be able to show how cool is a project. I am fine with removing the part concerning the Python module.

**Comment by gpotter2** [comment]
There is still

> On most systems, using Scapy is as simple as running the following commands:
> ```
git clone https://github.com/secdev/scapy
cd scapy
./run_scapy
```

A few lines below. I thought that was enough. We could move it above.

I also thought of adding an SVG animation that installs Scapy and start it. That could be quite fancy

**Comment by guedou** [comment]
Thanks. I did not see that. I still prefer to keep the `hands-on` section, but the animation seems to do the job.

Could you make a new _video_ by setting `$PS1` to "# " or "demo# ", install the `matplotlib` module, and run it on an host with IPv6 connectivity? That will be nicer without the errors messages =

Otherwise, it looks good to me. Thanks for this PR!

**Comment by gpotter2** [comment]
I've restored the shell part of the hands on.
I'll work my way through the recording.

I definetly agree with the warnings, but isn't it going to be confusing when we `cd` a folder that isn't shown ? I'm using asciicast - termtosvg bindings, which mean I have a text file I can easily edit and regenerate the SVG. We could do anything if required

**Comment by guedou** [comment]
I like clean prompts for demonstrations. We can use `PS1="\w$"` instead.

asciinema is a really good altetnative. The v2.0 format can easily be edited manually.

File: README.md [hunk 1]
```diff
@@ -56,43 +61,21 @@ Received 2 packets, got 1 answers, remaining 0 packets
...
-notebooks.
-
-The [documentation](http://scapy.readthedocs.io/en/latest/) contains more
+The [documentation](https://scapy.readthedocs.io/en/latest/) contains more
 advanced use cases, and examples.
 
-## Installation
+Other useful resources:
+
+- [Scapy in 20 minutes](https://github.com/secdev/scapy/blob/master/doc/notebooks/Scapy%20in%2015%20minutes.ipynb)
```
Comment (p-l-) [inline README.md]: Codacy complains about missing newlines after each list element.

File: README.md [hunk 2]
```diff
@@ -56,43 +61,21 @@ Received 2 packets, got 1 answers, remaining 0 packets
...
+-  [Interactive tutorial](https://scapy.readthedocs.io/en/latest/usage.html#interactive-tutorial) (part of the documentation)
```
Comment (p-l-) [inline README.md]:
```suggestion
- [Interactive tutorial](https://scapy.readthedocs.io/en/latest/usage.html#interactive-tutorial) (part of the documentation)
```

File: README.md (hunk 1 above)
Comment (gpotter2) [reply README.md]: Should be fixed by now, thanks for the call

**Comment by guedou** [comment]
@p-l- it looks good to me. Do you agree?

**Pull request merged**